### PR TITLE
Add the v1 (stable) endpoint.

### PR DESCRIPTION
**Pull request by lukesneeringer**
This PR adds the `v1` endpoint to video intelligence _and_ marks the library as GA.

This PR should *not* be merged until just before the November 15 rollout.

### CODE REVIEW

File: samples/README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 # Google Cloud Video Intelligence API: Node.js Samples
 
-[![Open in Cloud Shell][shell_img]][shell_link]
+[![Build](https://storage.googleapis.com/.svg)]()
```
Comment (stephenplusplus) [inline samples/README.md]: Missing image link.

File: src/v1/doc/google/cloud/videointelligence/v1/doc_video_intelligence.js [hunk 2]
```diff
@@ -0,0 +1,652 @@
+// Copyright 2017, Google LLC All rights reserved.
```
Comment (stephenplusplus) [inline src/v1/doc/google/cloud/videointelligence/v1/doc_video_intelligence.js]: Do we still need these dummy doc files?

**Comment by stephenplusplus** [comment]
I could be mistaken, but it looks like the README changes might be backwards, like maybe from an outdated version of the generator?

File: src/v1/doc/google/cloud/videointelligence/v1/doc_video_intelligence.js (hunk 2 above)
Comment (lukesneeringer) [reply src/v1/doc/google/cloud/videointelligence/v1/doc_video_intelligence.js]: I think so.

**Comment by lukesneeringer** [comment]
> I could be mistaken, but it looks like the README changes might be backwards, like maybe from an outdated version of the generator?

Yeah, I think so. Fixed.

**Comment by lukesneeringer** [comment]
Staging release notes in the usual location (so they are ready next week).

File: samples/README.md [hunk 3]
```diff
@@ -29,21 +29,16 @@ View the [source code][video_0_code].
...
 Commands:
-  analyze.js faces <gcsUri>        Analyzes faces in a video stored in Google Cloud Storage using the Cloud Video
-                                   Intelligence API.
-  analyze.js shots <gcsUri>        Analyzes shot angles in a video stored in Google Cloud Storage using the Cloud Video
-                                   Intelligence API.
-  analyze.js labels-gcs <gcsUri>   Labels objects in a video stored in Google Cloud Storage using the Cloud Video
-                                   Intelligence API.
-  analyze.js labels-file <gcsUri>  Labels objects in a video stored locally using the Cloud Video Intelligence API.
-  analyze.js safe-search <gcsUri>  Detects explicit content in a video stored in Google Cloud Storage.
+  faces <gcsUri>        Analyzes faces in a video stored in Google Cloud Storage using the Cloud Video Intelligence API.
```
Comment (stephenplusplus) [inline samples/README.md]: I think this is out of date. The other repos use the `{filename} {command} {arguments}` notation. Example: https://github.com/googleapis/nodejs-storage/blob/master/samples/README.md

Comment (lukesneeringer) [reply samples/README.md]: I agree that looks wrong, but I also _just_ generated it yesterday. Will look into it when I have a chance (definitely this week as this goes out Wednesday).

File: src/index.js [hunk 4]
```diff
@@ -47,6 +51,10 @@ const gapic = Object.freeze({
  *
  * - `VideoIntelligenceServiceClient` - Reference to
  *   {@link v1beta2.VideoIntelligenceServiceClient}
```
Comment (callmehiphop) [inline src/index.js]: I believe this link needs to be updated to `v1`

Comment (lukesneeringer) [reply src/index.js]: They should both be there.

File: samples/README.md (hunk 3 above)
Comment (lukesneeringer) [reply samples/README.md]: I just checked this on-branch and it is correct. If you look above you see the `node analyze.js --help` part. (Note that this is the sample readme and not the main one).

File: src/index.js (hunk 4 above)
Comment (lukesneeringer) [reply src/index.js]: I misunderstood this. You are right. Fixed.

**Comment by lukesneeringer** [comment]
@callmehiphop Is this good to go now?

**Comment by callmehiphop** [comment]
@lukesneeringer yepyep!

File: src/index.js [hunk 5]
```diff
@@ -82,16 +90,29 @@ const gapic = Object.freeze({
...
  * @property {constructor} VideoIntelligenceServiceClient
  *   Reference to {@link v1beta2.VideoIntelligenceServiceClient}
  */
 module.exports.v1beta2 = gapic.v1beta2;
+/**
+ * @type {object}
+ * @property {constructor} VideoIntelligenceServiceClient
+ *   Reference to {@link v1beta2.VideoIntelligenceServiceClient}
+ */
+module.exports.v1beta2 = gapic.v1beta2;
```
Comment (michaelbausor) [inline src/index.js]: @lukesneeringer Is this meant to be duplicated? (with line 109)

**Pull request merged**